static/src/js/components/BoundingRectangleField/__tests__/BoundingRectangleField.test.jsx

```diff
@@ -32,17 +42,19 @@ describe('BoundingRectangleField', () => {
     test('renders the fields', () => {
       setup()
 
-      const west = screen.getByLabelText('West')
-      expect(west).toHaveValue(-179)
+      const spinbuttons = screen.getAllByRole('spinbutton')
```

Comment:
Why was this changed? This is a less descriptive way to find each field. Having to look at array indexes (`spinbuttons[0]`) is harder to maintain and more prone to errors than finding a field by label text.

Reply:
because the other ones weren't working and the spin buttons don't have names. They are harder to target without test-ids. Any other suggestions? I tried screen.getByText and screen.getByRole('spinbutton', ( id: 'north')} but neither did the trick. I'll do some more digging on my end and see if I can find something better.

Reply:
There is still a label that is supposed to be attached to the input fields, but the `id` you are passing in to `CustomWidgetWrapper` broke that link. 

Previously the label was setup like this 
```
<label htmlFor="north-coordinate">North</label>
```

But you are passing in `name` to the `id` prop, and `CustomWidgetWrapper` uses `htmlFor={id}` in the label.

Aside from the tests, you should be able to see is broken by clicking on the label text in your browser. That should focus the correct input field, which is why we want our test to use `getByLabelText`, to ensure that our tests properly interact with the application in a way a user would.